src/components/ui/Menubar/fragments/MenubarTrigger.tsx

```
@@ -0,0 +1,42 @@
+  className?: string;
+} & MenuPrimitiveProps.Trigger;
+
+const MenubarTrigger = ({ children, className }:MenubarTriggerProps) => {
+    const context = React.useContext(MenubarContext);
+    if (!context) {
+        console.log('MenubarTrigger should be used in the MenubarRoot');
+        return null;
+    }
+    const { rootClass } = context;
+
+    const menuContext = React.useContext(MenubarMenuContext);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)